Point to all service definitions in the docs. (#702)
cachix/devenv: docs/services.md

{%
  include-markdown "services-all.md"
%}

> You can find the definitions of all services [here](https://github.com/cachix/devenv/tree/main/src/modules/services).

Reviewer: ```suggestion
You can find all supported options for services [here](https://devenv.sh/reference/options/#servicesadminerenable).
```

Author: Ooh that's even better! Thanks!
